examples/heal/README.md

```diff
@@ -19,3 +20,14 @@ To run any heal example follow steps for [Basic NSM setup](../basic)
  - [Remote NSMgr restart ip](./remote-nsmgr-restart-ip)
  - [Remote Forwarder death ip](./remote-forwarder-death-ip)
  - [Remote NSE death ip](./remote-nse-death-ip)
+
+ Two components:
+ - [Local NSMgr + Remote NSMgr](./local-nsmgr-remote-nsmgr)
+ - [Local NSMgr + Local Forwarder](./local-nsmgr-local-forwarder)
+ - [Local Forwarder + Remote Forwarder](./local-forwarder-remote-forwarder)
+ - [Registry + Local Endpoint](./registry-local-endpoint)
```

Comment:
Why we don't test remote endpoint restarts?

Reply:
We need to add it
Done